### DISCUSSION TITLE
Recommended computer specifications for QCxMS simulation

### DISCUSSION THREAD

**Discussion by YuanChihChen** [qcxms/QCxMS#84] [Q&A]
Hello,
I currently utilize my laptop to operate the QCxMS program, employing an AMD 4800HS 8-core CPU and 16 GB RAM. However, the extensive analysis times have restricted my ability to further utilize QCxMS in my research. I want to ask what the recommended computer specifications are when simulating the CID fragmentation of the small molecules with a mass of approximately 300 Da.

Additionally, we also aim to explore simulating the fragmentation of peptides with a mass of around 4000 Da. I'm curious whether it is feasible to perform such simulations on a standard workstation equipped with an 8 to 32-core CPU.

Thank you for the help,
YC Chen

**Comment by tobigithub** [comment]
We have run QCxMS on 8 core CPUs up to 10k HPC clusters with many molecules. I think its worthwhile to note that for such a large molecule (4000 Da), the first MD trajectory that does the sampling will be one of the speed limiting factors, if that is even possible. Beyond that, single CPU speed would be the most important and then if you run lets say 64 trajectories I would only run that on a cluster or minimum 128 core CPU. For smaller molecules like 300 Da you might be OK with 32 core CPU.
